bleak/backends/bluezdbus/client.py

```diff
@@ -365,6 +365,11 @@ async def pair(self, *args, **kwargs) -> bool:
             Boolean regarding success of pairing.
 
         """
+        if self._bus is None:
+            self._bus = await MessageBus(
```

Comment:
maybe this could be using a central function that connects the bus in all cases with the same code?

Reply:
Makes sense 😄